### ISSUE TITLE
import bug for BIBTEX multiple keywords tag

### ISSUE THREAD

**Issue by g10996694** [JabRef/jabref#483]
In the bibtex exported from sciencedirect, there are often multiple keywords tags. In fact, every tag contains one keyword. 
But jabref takes only the first one, the rest are ignored. 
Perhaps parsing loop would help to load all keywords ?

**Comment by simonharrer** [comment]
Hm, hard to say if this is really a bug or a feature. The problem here is that sciencedirect produces a wrong bibtex format.

**Comment by g10996694** [comment]
@simonharrer 
Yes, the bibtex format export from ScienceDirect does not respect the rules of BIBTEX.
That's why I use generally the RIS export instead of BIBTEX export.
Just find this problem, it's you to decide :)

**Comment by simonharrer** [comment]
Probably something for the JabRef developers to discuss. :)

**Comment by tobiasdiez** [comment]
Something similar is already done for the author and editor fields. 
To cite the BibtexParser:
```` java
                // The following hack enables the parser to deal with multiple
                // author or
                // editor lines, stringing them together instead of getting just
                // one of them.
                // Multiple author or editor lines are not allowed by the bibtex
                // format, but
                // at least one online database exports bibtex like that, making
                // it inconvenient
                // for users if JabRef didn't accept it.
                if ("author".equals(key) || "editor".equals(key)) {
                    entry.setField(key, entry.getField(key) + " and " + content);
                }
````

**Comment by lenhard** [comment]
This is now fixed in master with e1fcdd35e919b55599f9d07d0097cd3eca2eaf60
